api/copy_chunk_experimental.md

```diff
@@ -18,8 +18,24 @@ do not use any experimental features in production.
 |`source_node`|NAME|Data node where the chunk currently resides|
 |`destination_node`|NAME|Data node where the chunk is to be copied|
 
+### Required settings
+
+For `copy_chunk` to work, the `wal_level` must be set to `logical` or
+higher on data nodes. Also consider increasing `max_wal_senders` and
+`max_replication_slots` if many chunks are copied or moved in
+parallel.
+
+### Failures
+
+A failed copy operation will hold a replication slot open and needs to
```

Comment:
not just repl slot, but it might have created a chunk on the destination node, created a subscription. So all of this needs to be cleaned up and is done via the cleanup function.